apis/management.cattle.io/v3/windows_types.go

```diff
@@ -1,8 +1,6 @@
 package v3
 
 type WindowsSystemImages struct {
-	// Windows nginx-proxy image
-	NginxProxy string `yaml:"nginx_proxy" json:"nginxProxy,omitempty"`
 	// Kubernetes binaries image
```

Comment:
Will it introduce some issue if we delete the nginx proxy here but we keep it in the `NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),`

Reply:
If we don't consider upgrading from 2.1, I think we don't need to keep it.